### PR TITLE
SUBMARINE-322. Add yarn runtime examples in installation guide.

### PR DESCRIPTION
**Pull request by yuanzac**
### What is this PR for?
Instead of yarn native service, yarn runtime is used by default since submarine 0.3.0,
The guide of how to a run yarn runtime job should be added in the submarine installation guide. 

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-322

### How should this be tested?
https://travis-ci.org/yuanzac/hadoop-submarine/builds/630008273

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md [hunk 1]
```diff
@@ -55,13 +55,22 @@ Refer to [Write Dockerfile](WriteDockerfileTF.md) to build a Docker image or use
...
+The file, named submarine-site.xml, is in the path of ${SUBMARINE_HOME}/conf.
+
 ### Run standalone training
 
 ```
-SUBMARINE_VERSION=0.2.0
-CLASSPATH=`path-to/hadoop classpath --glob`:path-to/hadoop-submarine-core-${SUBMARINE_VERSION}.jar:
-path-to/hadoop-submarine-yarnservice-runtime-${SUBMARINE_VERSION}.jar:path-to/hadoop-submarine-tony-
-runtime-${SUBMARINE_VERSION}.jar \
+SUBMARINE_VERSION=0.3.0
```
Comment (liuxunorg) [inline docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: SUBMARINE_VERSION=0.3.0-SNAPSHOT ??

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md [hunk 2]
```diff
@@ -80,10 +89,9 @@ Explanations:
 ### Run distributed training
 
 ```
-SUBMARINE_VERSION=0.2.0
-CLASSPATH=`path-to/hadoop classpath --glob`:path-to/hadoop-submarine-core-${SUBMARINE_VERSION}.jar:
-path-to/hadoop-submarine-yarnservice-runtime-${SUBMARINE_VERSION}.jar:path-to/hadoop-submarine-tony-
-runtime-${SUBMARINE_VERSION}.jar \
+SUBMARINE_VERSION=0.3.0
```
Comment (liuxunorg) [inline docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: SUBMARINE_VERSION=0.3.0-SNAPSHOT ??

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md [hunk 3]
```diff
@@ -182,8 +190,9 @@ When using YARN native service runtime, you can view multiple job training histo
 ```shell
 # Cleanup previous tensorboard service if needed
 
-SUBMARINE_VERSION=0.2.0
-CLASSPATH=`path-to/hadoop classpath --glob`:path-to/hadoop-submarine-core-${SUBMARINE_VERSION}.jar:path-to/hadoop-submarine-yarnservice-runtime-${SUBMARINE_VERSION}.jar:path-to/hadoop-submarine-tony-runtime-${SUBMARINE_VERSION}.jar \
+SUBMARINE_VERSION=0.3.0
```
Comment (liuxunorg) [inline docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: SUBMARINE_VERSION=0.3.0-SNAPSHOT ??

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md (hunk 1 above)
Comment (yuanzac) [reply docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: Done

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md (hunk 3 above)
Comment (yuanzac) [reply docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: Done

File: docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md (hunk 2 above)
Comment (yuanzac) [reply docs/helper/RunningDistributedCifar10TFJobsWithYarnService.md]: Done

**Comment by liuxunorg** [comment]
@yuanzac Thank you contribution.
Will merge if no more comments.

**Pull request closed without merging**